src/Adapter/Alias/Repository/AliasRepository.php

```diff
@@ -219,4 +245,48 @@ public function searchSearchTerms(string $searchPhrase, ?int $limit = null): arr
             ->executeQuery()
             ->fetchAllAssociative();
     }
+
+    /**
+     * @param array $searchTerms
+     *
+     * @return array{
+     *   array{
+     *     id_alias: int,
+     *     search: string,
+     *     alias: string,
+     *     active: bool,
+     *   }
+     * }
+     */
+    public function getAliasesBySearchTerms(array $searchTerms): array
+    {
+        $qb = $this->connection->createQueryBuilder()
+            ->addSelect('a.id_alias, a.search, a.alias, a.active')
+            ->from($this->dbPrefix . 'alias', 'a')
+            ->where('a.search IN (:searchTerms)')
+            ->setParameter('searchTerms', $searchTerms, ArrayParameterType::STRING)
+            ->addOrderBy('a.search', 'ASC')
+            ->addOrderBy('a.alias', 'ASC')
+        ;
+
+        return $qb->executeQuery()->fetchAllAssociative();
+    }
+
+    /**
+     * @param string $searchTerm
+     *
+     * @return string[]
+     */
+    public function getAliasesAlreadyInUseNotForSearchTerm(string $searchTerm): array
```

Comment:
```suggestion
    private function getAliasesAlreadyInUseNotForSearchTerm(string $searchTerm): array
```
Seems like it's only used internally no?